### PR TITLE
implementation of issue #289

### PR DESCRIPTION
**Pull request by ricardodemauro**
Hello guys,

I just implement the issue 289.

I notice that in appsettings.docker.json there is a begin of implementation of azure storage (not sure here). Anyway, I leave it as is and implement the changes in appsettings.json.

I also change the "SimplCommerce.WebHost.csproj" including the WindowsAzure.Storage package. I didn't find the problem of just add in the new package. The solution was to add in the webhost project.

The configuration and usage it's the same of AWS implementation.

Hope you find useful!

Thanks

### CODE REVIEW

File: src/SimplCommerce.WebHost/appsettings.docker.json [hunk 1]
```diff
@@ -11,10 +11,18 @@
     }
   },
   "Storage": {
```
Comment (thiennn) [inline src/SimplCommerce.WebHost/appsettings.docker.json]: Please help to remove this section

File: src/SimplCommerce.WebHost/SimplCommerce.WebHost.csproj [hunk 2]
```diff
@@ -19,6 +19,10 @@
     <ProjectReference Include="..\Modules\SimplCommerce.Module.Localization\SimplCommerce.Module.Localization.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (thiennn) [inline src/SimplCommerce.WebHost/SimplCommerce.WebHost.csproj]: The host should not depend on WindowsAzure.Storage package.

**Comment by ricardodemauro** [comment]
Hello Thiennn,

I notice that when the below code is executed, the dll WindowsAzure.Storage is blowing the "Could not assembly....."

foreach (var file in binFolder.GetFileSystemInfos("*.dll", SearchOption.AllDirectories))
{
    Assembly assembly;
        try
        {
             assembly = AssemblyLoadContext.Default.LoadFromAssemblyPath(file.FullName);
         }
         catch (FileLoadException)
         {

That was the main reason why I added WindowsAzure.Storage in WebHost project.
Not sure what is causing this error.

I'm thinking maybe use ILMerge or another tool to generate a single dll. Maybe this is a possible solution to our problem. I'll investigate.

What do you think about that?

**Comment by thiennn** [comment]
Please check whether the WindowsAzure.Storage dlls are included in the WebHost/Modules/SimplCommerce.Module.StorageAzureBlob/bin.

.NET Core by default doesn't include package dependencies in the bin folder on build. The temporary workaround is adding <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies> will change this default behavior. The downside of this is many ASP.NET Core dlls are also included.

**Comment by ricardodemauro** [comment]
Hi @thiennn,

Sorry for the very late response. I'll implement a newer version of this issue.
Can you delete this pull request?

Thanks,

**Pull request closed without merging**